CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
 - signal-hook update
 - Make windows raw_mode act on CONIN$
 - Added From<(u8, u8, u8)> Trait to Color::Rgb Enum
+- Added `crossterm::tty::IsTty` Trait to easily determine whether a output instance is a terminal teletype.
```

Comment:
Why this? The trait was added in a previous version.

Reply:
Sorry :D

My rust dev tools didn't pick up the `tty` module until I updated to `0.17.6` from `0.17.5`. Naturally, I assumed that it must have been added in `0.17.6`, but couldn't find a entry in this specific changelog version and promptly decided to add it.

Retrospectively that was a little hasty. Looks like it was just a small derp of my rust dev tools...